lib/libc/common/Kconfig

```diff
@@ -5,3 +5,34 @@ config COMMON_LIBC_TIME
 	bool
 	help
 	  common implementation of time().
+
+config COMMON_LIBC_MALLOC
+	bool
+	help
+	  common implementation of malloc family
```

Comment:
_(Continuing from the comment above)_

This needs to have a prompt so that it can be configured in the menuconfig as well as in the conf files (e.g. `CONFIG_COMMON_LIBC_MALLOC=n`.

Reply:
Yup. All fixed.